app/src/main/res/layout/scan_text_view_accent.xml

```diff
@@ -26,6 +26,10 @@
         <variable
             name="initialValue"
             type="java.lang.String" />
+
```

Comment:
could we merge scan_text_view and scan_text_view_accent and use style provider for the differences?

Reply:
These two layouts were used in the ScanTextView class that I've removed. So I will remove them too.

Reply:
👏🏽